scripts/release/make-release.py

```diff
@@ -71,9 +64,8 @@ def write_release_log(message):
 def commit_tag_and_push():
     run("git add RELEASE_LOG.md")
     run("git commit -m \"{msg}\"".format(msg=release_message))
-    print("Tagging and pushing...")
+    print("Tagging...")
     tag(new_tag, release_message)
-    push()
```

Comment:
Alex and I decided it would be better if the tool auto-committed, but didn't push, so you have a last minute chance to review and amend if need be.

Reply:
:+1: this seems a better idea